<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RahanTalletus</class>
 <widget class="QDialog" name="RahanTalletus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Rahan talletus</string>
  </property>
  <widget class="QSpinBox" name="seteli10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/kuvat/10e.jpg);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="seteli20">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/kuvat/20e.jpg)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="seteli50">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>330</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/kuvat/50e.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Talleta</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="seteli100">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/kuvat/100e.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 â‚¬</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="peruutaButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
